### DISCUSSION TITLE
FIDO2 certification

### DISCUSSION THREAD

**Discussion by huaracheguarache** [solokeys/solo2#142] [Q&A]
According to the Kickstarter update from 20th of July you were targeting the 20-23 September Interop Event to receive a FIDO2 certification. What's the latest news on this?

**Comment by huaracheguarache** [comment]
@conorpp @nickray @0x0ece Since the last firmware update things have been pretty quiet with no new commits, but plenty of people are still having issues with their keys on various services (see #96, #141, #137). I backed the project early on by buying a large amount of keys for both myself and my family. While I'm personally using my keys, I'm hesitant to provide them to other family members due to the many issues people have reported. I understand that the project is unprofitable and that the devs have other priorities, but a lot of us are very frustrated that we have not gotten a working product that we paid for.

**Comment by theerror** [comment]
Well, just come here for same reason. I've rely on certification a lot when bought. Actually, I thought it's kind of already done thing, that certification is in place already and what was my surprise I wasn't able to reach L1 cert pass needed in our government services. I can see solokey on FIDO alliance's site, however look, that it's not fully certificated yet? Is there some roadmap? 

![image](https://user-images.githubusercontent.com/4508328/196128733-e9c8bfb7-9943-4a16-8df7-102407be66c8.png)

**Reply by theerror** [reply]
well, now come to my mind - isn't that the old (v1) one?

**Reply by needs-coffee** [reply]
@theerror - yeah i think that is the original one not the solo2

**Reply by goebbe** [reply]
Well, the naming is ambiguous. 
The entry is for Solo, Version 2.0. Isn't this the same as solo v2 or solo2?  Or does the Version refer to something different?
https://fidoalliance.org/certification/fido-certified-products/

**Reply by hase77** [reply]
It's not ambiguous, Solo and Solo v2 are different products. Version is FIDO version 1.0 or 2.0.

**Reply by goebbe** [reply]
@hase77 thanks for clarifying.

**Comment by hase77** [comment]
And please not only Fido certification but also certification with Microsoft (https://learn.microsoft.com/en-us/azure/active-directory/authentication/concept-fido2-hardware-vendor).

Best regards
Sebastian

**Reply by msiqx9** [reply]
What is the purpose of this certification given that Solo v1 and Solo v2 already work with Microsoft on FIDO2?

**Reply by hase77** [reply]
MS FIDO2 Certification has nothing to do with the login to MS webservices. It is used for Azure, AD, Windows Login etc.. At this point a Solo2 for example is useless for Windows Login unless you are using Azure AD and bypass the certificaton requirement. Which works flawless with an Y***-Key.

And it is very sad that there is no reaction to the inital post with the request of an status update of the FIDO2 certification.

**Comment by msiqx9** [comment]
There seems to be some code [commit](https://github.com/solokeys/solo2/commit/bf0e51fc44daedb7b8f9edd6fb9e27b00d2fd319) yesterday so maybe the certification may come.
